src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/retriever/Frs10xAccountsBoxRetriever.scala

```diff
@@ -16,12 +16,19 @@
 
 package uk.gov.hmrc.ct.accounts.frs10x.retriever
 
-import uk.gov.hmrc.ct.accounts.frs10x.boxes.{AC8081, AC8082, AC8083, AC8088}
-import uk.gov.hmrc.ct.accounts.retriever.AccountsBoxRetriever
-import uk.gov.hmrc.ct.box.retriever.FilingAttributesBoxValueRetriever
+import uk.gov.hmrc.ct.{CoHoAccountsApprovalRequired, HmrcAccountsApprovalRequired}
+import uk.gov.hmrc.ct.accounts.frs10x.boxes._
+import uk.gov.hmrc.ct.accounts.retriever.{AccountsApprovalRequiredBoxRetriever, AccountsBoxRetriever}
+import uk.gov.hmrc.ct.box.retriever.{BoxRetriever, FilingAttributesBoxValueRetriever}
 
-trait Frs10xAccountsBoxRetriever extends AccountsBoxRetriever {
-  self: FilingAttributesBoxValueRetriever =>
+abstract class Frs10xAccountsBoxRetriever(filingAttributesBoxRetriever: FilingAttributesBoxValueRetriever)
```

Comment:
So is the idea that eventually we'll be able to pass in all the retrievers here? Which replaces the the inherited traits below? Will have to have a talk to to you on Slack about.